src/core/utils/formatterUtils/formatterUtils.mdx

```diff
@@ -114,6 +117,115 @@ of the total supply.
+
+```typescript
+import { DateFormat, formatterUtils } from '@aragon/ods';
+
+const formattedDate = formatterUtils.formatDate(date, { format: DateFormat.YEAR_MONTH });
+```
+
+### YEAR_MONTH_DAY_TIME
+
+The `YEAR_MONTH_DAY_TIME` format displays full dates with times and uses a "friendly" format when the date is yesterday,
+today or tomorrow.
+
+**Examples**:
+
+<FormatExampleTable
+    values={[
+        '2009-01-02T10:10:41.049Z',
+        '2023-10-23T15:33:12.049Z',
+        DateTime.utc().plus({ hours: 3 }).toISO(),
+        DateTime.utc().minus({ hours: 23 }).toISO(),
+        DateTime.utc().plus({ hours: 24 }).toISO(),
+    ]}
+    formats={[DateFormat.YEAR_MONTH_DAY_TIME]}
+    isDate={true}
+/>
+
+### YEAR_MONTH_DAY
+
+The `YEAR_MONTH_DAY` format displays full dates and, similiarly to the `YEAR_MONTH_DAY_TIME` format, uses a "friendly"
+format when the date is yesterday, today or tomorrow.
+
+**Examples**:
+
+<FormatExampleTable
+    values={[
+        '2009-01-02T10:10:41.049Z',
+        '2023-10-23T15:33:12.049Z',
+        DateTime.utc().plus({ hours: 3 }).toISO(),
+        DateTime.utc().minus({ hours: 23 }).toISO(),
+        DateTime.utc().plus({ hours: 24 }).toISO(),
+    ]}
+    formats={[DateFormat.YEAR_MONTH_DAY]}
+    isDate={true}
+/>
+
+### YEAR_MONTH
+
+The `YEAR_MONTH` format only displays the year and month of the specified date and never uses a "friendly" format.
+
+**Examples**:
+
+<FormatExampleTable
+    values={[
+        '2009-01-02T10:10:41.049Z',
+        '2023-10-23T15:33:12.049Z',
+        DateTime.utc().plus({ hours: 3 }).toISO(),
+        DateTime.utc().minus({ hours: 23 }).toISO(),
+        DateTime.utc().plus({ hours: 24 }).toISO(),
+    ]}
+    formats={[DateFormat.YEAR_MONTH]}
+    isDate={true}
+/>
+
+### DURATION
+
+The `DURATION` format returns the date difference between the specified date and now. It uses the largest date unit that
+is greater than 1.
+
+**Examples**:
+
+<FormatExampleTable
+    values={[
+        '2009-01-02T10:10:41.049Z',
+        '2023-10-23T15:33:12.049Z',
+        DateTime.utc().minus({ hours: 23 }).toISO(),
+        DateTime.utc().minus({ seconds: 1 }).toISO(),
+        DateTime.utc().plus({ hours: 25 }).toISO(),
+        DateTime.utc().plus({ months: 4 }).toISO(),
+    ]}
+    formats={[DateFormat.DURATION]}
+    isDate={true}
+/>
+
+### RELATIVE
+
+Similiarly to the `DURATION` format, the `RELATIVE` format returns the date difference between the specified date and
+now and displays if the date is in the past or in the future.
+
+**Examples**:
+
+<FormatExampleTable
+    values={[
+        '2009-01-02T10:10:41.049Z',
+        '2023-10-23T15:33:12.049Z',
+        DateTime.utc().minus({ hours: 23 }).toISO(),
+        DateTime.utc().minus({ seconds: 1 }).toISO(),
+        DateTime.utc().plus({ hours: 25 }).toISO(),
+        DateTime.utc().plus({ months: 4 }).toISO(),
+    ]}
+    formats={[DateFormat.RELATIVE]}
+    isDate={true}
```

Comment:
I wonder if we should format as '9 months from now' instead of start with 'In 9 months' so that we keep the number at the top of the string

Reply:
Good point, but I believe this is not supported by luxon :/